treeherder/model/models.py

```diff
@@ -601,13 +601,17 @@ class JobDetail(models.Model):
     There can be (and usually is) more than one of these associated with
     each job
     '''
-    MAX_FIELD_LENGTH = 512
+    MAX_FIELD_LENGTHS = {
+        "title": 70,
+        "value": 125,
+        "url": 512
+    }
```

Comment:
I have to say this is pretty verbose. I think we'd be better off using Django's _meta API, which seems to be more or less supported in recent versions:

https://docs.djangoproject.com/en/1.9/ref/models/meta/

In this case you can set the field lengths inline and just use something like:

```py
JobDetail._meta.get_field('title').max_length
```

to get them